platform_versioned_docs/version-24.2/compute-envs/azure-batch.mdx

```diff
@@ -311,7 +438,7 @@ Create a manual Seqera Azure Batch compute environment:
     Configuration settings in this field override the same values in the pipeline repository `nextflow.config` file. See [Nextflow config file](../launch/advanced.mdx#nextflow-config-file) for more information on configuration priority. 
     :::
     :::info
-    To use managed identities, Platform requires Nextflow version 24.10.0 or later.
+    Nextflow must be version 24.10.0 or later to use managed identities.
```

Comment:
@adamrtalbot This contradicts the earlier admonitions that states 24.06.0-edge is required. This admonition also seems to be in the wrong place, as nothing relates to managed identities here